templates/includes/sidebar.html

```diff
@@ -0,0 +1,14 @@
+<ul>
+  <li>
+    <a href="#">Home page</a>
+  </li>
+  <li>
+    <a href="#">Manufacturers</a>
+  </li>
+  <li>
+    <a href="#">Cars</a>
+  </li>
+  <li>
+    <a href="#">Drivers</a>

```

Comment:
The href attributes in the anchor tags are set to `#`, which is a placeholder. Replace these with actual URLs or use Django's `{% url %}` template tag to link to the appropriate views.